`test_pylint.py` throws `Exception: Invalid font prefix "mdi"` when run alone

### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [ ] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

This is likely to be something weird and local, so I'm asking just in case you've seen it before and have an idea of what might be causing it.  If not, feel free to close it.

I've just built 6.0.2 for Debian (thanks for the great work on this!).  The tests all run fine (though I haven't tried re-including those which failed previously), except for a couple.  The weird one is this:

```
_________ ERROR collecting spyder/plugins/pylint/tests/test_pylint.py __________
spyder/plugins/pylint/tests/test_pylint.py:34: in <module>
    from spyder.plugins.pylint.plugin import Pylint
spyder/plugins/pylint/plugin.py:25: in <module>
    from spyder.plugins.pylint.confpage import PylintConfigPage
spyder/plugins/pylint/confpage.py:16: in <module>
    from spyder.plugins.pylint.main_widget import (MAX_HISTORY_ENTRIES,
spyder/plugins/pylint/main_widget.py:90: in <module>
    class CategoryItem(QTreeWidgetItem):
spyder/plugins/pylint/main_widget.py:102: in CategoryItem
    'icon': ima.icon("convention")
spyder/utils/icon_manager.py:462: in icon
    return qta.icon(*args, **kwargs)
/usr/lib/python3/dist-packages/qtawesome/__init__.py:234: in icon
    return _instance().icon(*names, **kwargs)
/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:425: in icon
    parsed_options.append(self._parse_options(specific_options,
/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:477: in _parse_options
    prefix, chars = self._get_prefix_chars(names)
/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:511: in _get_prefix_chars
    raise Exception(error)
E   Exception: Invalid font prefix "mdi"
__ ERROR collecting spyder/plugins/pylint/tests/test_pylint_config_dialog.py ___
spyder/plugins/pylint/tests/test_pylint_config_dialog.py:25: in <module>
    from spyder.plugins.pylint.plugin import Pylint
spyder/plugins/pylint/plugin.py:25: in <module>
    from spyder.plugins.pylint.confpage import PylintConfigPage
spyder/plugins/pylint/confpage.py:16: in <module>
    from spyder.plugins.pylint.main_widget import (MAX_HISTORY_ENTRIES,
spyder/plugins/pylint/main_widget.py:90: in <module>
    class CategoryItem(QTreeWidgetItem):
spyder/plugins/pylint/main_widget.py:102: in CategoryItem
    'icon': ima.icon("convention")
spyder/utils/icon_manager.py:462: in icon
    return qta.icon(*args, **kwargs)
/usr/lib/python3/dist-packages/qtawesome/__init__.py:234: in icon
    return _instance().icon(*names, **kwargs)
/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:425: in icon
    parsed_options.append(self._parse_options(specific_options,
/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:477: in _parse_options
    prefix, chars = self._get_prefix_chars(names)
/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:511: in _get_prefix_chars
    raise Exception(error)
E   Exception: Invalid font prefix "mdi"
```

And it's weird because when I set up an lxc container manually to run it (which should be identical), it goes fine.  Do you have any clue what might be causing this error?  I don't recall ever seeing it before, and qtawesome is up-to-date.

Thanks!

Julian

Oh, it's even worse than that.  When I run pytest with `--ignore=spyder/plugins/pylint/tests/`, I get dozens (hundreds?) of errors caused by the Invalid font prefix issue, for example:
```
_______________ ERROR at setup of test_plugin_completion_gather ________________

>       lambda: ihook(item=item, **kwds), when=when, reraise=reraise
    )

/usr/lib/python3/dist-packages/flaky/flaky_pytest_plugin.py:146: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
spyder/plugins/completion/tests/conftest.py:75: in completion_plugin_wrap
    completions = CompletionPlugin(main_window, CONF)
spyder/plugins/completion/plugin.py:251: in __init__
    self.register_statusbar_widgets(plugin_loaded=False)
spyder/plugins/completion/plugin.py:467: in register_statusbar_widgets
    self.register_statusbar_widget(
spyder/plugins/completion/plugin.py:484: in register_statusbar_widget
    widgets_ids = container.register_statusbar_widgets(
spyder/plugins/completion/container.py:41: in register_statusbar_widgets
    statusbar = StatusBar(self)
spyder/plugins/completion/providers/languageserver/provider.py:206: in create_statusbar
    return LSPStatusWidget(parent, self)
spyder/plugins/completion/providers/languageserver/widgets/status.py:51: in __init__
    super().__init__(parent, show_spinner=True)
spyder/api/widgets/status.py:106: in __init__
    self._set_layout()
spyder/api/widgets/status.py:116: in _set_layout
    self._icon = self.get_icon()
spyder/plugins/completion/providers/languageserver/widgets/status.py:127: in get_icon
    return self.create_icon('lspserver.down')
spyder/api/widgets/mixins.py:690: in create_icon
    return ima.icon(name)
spyder/utils/icon_manager.py:462: in icon
    return qta.icon(*args, **kwargs)
/usr/lib/python3/dist-packages/qtawesome/__init__.py:234: in icon
    return _instance().icon(*names, **kwargs)
/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:425: in icon
    parsed_options.append(self._parse_options(specific_options,
/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:477: in _parse_options
    prefix, chars = self._get_prefix_chars(names)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <qtawesome.iconic_font.IconicFont object at 0x7f3f47cafd10>
names = ['mdi.alert', 'mdi.alert', 'mdi.alert', 'mdi.alert', 'mdi.alert', 'mdi.alert', ...]

    def _get_prefix_chars(self, names):
        chars = []
        for name in names:
            if '.' in name:
                prefix, n = name.split('.')
                if prefix in self.charmap:
                    if n in self.charmap[prefix]:
                        chars.append(self.charmap[prefix][n])
                    else:
                        error = 'Invalid icon name "{0}" in font "{1}"'.format(
                            n, prefix)
                        raise Exception(error)
                else:
                    error = 'Invalid font prefix "{0}"'.format(prefix)
>                   raise Exception(error)
E                   Exception: Invalid font prefix "mdi"

/usr/lib/python3/dist-packages/qtawesome/iconic_font.py:511: Exception
```

I'm stumped :(

--- a/spyder/plugins/preferences/widgets/configdialog.py
+++ b/spyder/plugins/preferences/widgets/configdialog.py
@@ -28,11 +28,11 @@ class ConfigDialog(SidebarDialog):
 
     # Constants
     TITLE = _("Preferences")
-    ICON = ima.icon('configure')
     MIN_WIDTH = 940 if MAC else (875 if WIN else 920)
     MIN_HEIGHT = 700 if MAC else (660 if WIN else 670)
 
     def __init__(self, parent=None):
+        self.ICON = ima.icon('configure')
         SidebarDialog.__init__(self, parent)
 
         # Attributes

